linera-views/src/backends/scylla_db.rs

```diff
@@ -11,6 +11,35 @@
 /// The limit is in reality 100. But we need one entry for the root key.
 const MAX_MULTI_KEYS: usize = 99;
 
+/// The maximal size of an operation on ScyllaDB seems to be 16M
+/// https://www.scylladb.com/2019/03/27/best-practices-for-scylla-applications/
+/// "There is a hard limit at 16MB, and nothing bigger than that can arrive at once
+///  at the database at any particular time"
+/// So, we set up the maximal size of 16M - 10K for the values and 10K for the keys
+/// We also arbitrarily decrease the size by 4000 bytes because an amount of size is
+/// taken internally by the database.
+const RAW_MAX_VALUE_SIZE: usize = 16762976;
+const MAX_KEY_SIZE: usize = 10240;
+const MAX_BATCH_TOTAL_SIZE: usize = 16773216;
+
+/// The RAW_MAX_VALUE_SIZE is the maximum size on the ScyllaDb storage.
+/// However, the value being written can also be the serialization of a SimpleUnorderedBatch
+/// Therefore the actual MAX_VALUE_SIZE might be lower.
+/// At the maximum the key_size is 1024 bytes (see below) and we pack just one entry.
+/// So if the key has 1024 bytes this gets us the inequality
+/// 1 + 1 + 1 + serialized_size(MAX_KEY_SIZE)? + serialized_size(x)? <= RAW_MAX_VALUE_SIZE
+/// and so this simplifies to 1 + 1 + (2 + 10240) + (4 + x) <= RAW_MAX_VALUE_SIZE
+/// (we write 4 because get_uleb128_size(RAW_MAX_VALUE_SIZE) = 4)
+/// and so to a maximal value of 408569;
```

Comment:
I don't understand how you arrive at 408569?

Reply:
It was a copy paste error.